### ISSUE TITLE
`Expected name as dictionary key` Exception

### ISSUE THREAD

**Issue by timjk** [UglyToad/PdfPig#145]
Hi we've seen the above exception in our application logs unable to reproduce atm.

```
_UglyToad.PdfPig.Exceptions.PdfDocumentFormatException: Expected name as dictionary key, instead got: SAPinfoStart TOA_DARA
at UglyToad.PdfPig.Tokenization.DictionaryTokenizer.ConvertToDictionary(IReadOnlyList`1 tokens)
at UglyToad.PdfPig.Tokenization.DictionaryTokenizer.TryTokenize(Byte currentByte, IInputBytes inputBytes, IToken& token) 
at UglyToad.PdfPig.Tokenization.Scanner.CoreTokenScanner.MoveNext()
at UglyToad.PdfPig.Tokenization.Scanner.PdfTokenScanner.MoveNext()
at UglyToad.PdfPig.Tokenization.Scanner.PdfTokenScanner.Get(IndirectReference reference) 
at UglyToad.PdfPig.Parser.Parts.DirectObjectFinder.Get[T](IndirectReference reference, IPdfTokenScanner scanner) 
at UglyToad.PdfPig.Parser.DocumentInformationFactory.Create(IPdfTokenScanner pdfTokenScanner, TrailerDictionary trailer) 
at UglyToad.PdfPig.Parser.PdfDocumentFactory.OpenDocument(IInputBytes inputBytes, ISeekableTokenScanner scanner, IContainer container, Boolean isLenientParsing, String password) 
at UglyToad.PdfPig.Parser.PdfDocumentFactory.Open(IInputBytes inputBytes, ParsingOptions options) 
at document.packs.bff.Utils.PdfFileUtils.GetPdfProperties(IFormFile file)
```

**Comment by EliotJones** [comment]
Hi there, sorry to hear you're running into this error. I thought the `SAPinfoStart` might be relevant/a clue to the document producer so I googled it and this thread seems related https://answers.sap.com/answers/11694119/view.html

The error might be in:

```
/Producer (SAP NetWeaver 700 )
%SAPinfoStart TOA_DARA
```

The comment in the dictionary should already be handled but I'll add a test around it to confirm, if not that's likely to be the bug.

**Comment by EliotJones** [comment]
I've checked this works and it seems it was fixed in version 0.1.0. If you're running 0.1.0 now you shouldn't see this error again. Let me know if you want to keep this issue open in case you find a document that reproduces the error.

**Comment by timjk** [comment]
Thanks @EliotJones we're not running 0.1.0 will update to that version & create another issue if we see it again.
